.github/workflows/testBuild.yml

```diff
@@ -140,39 +140,7 @@ jobs:
           APPLE_ID_PASSWORD: ${{ secrets.APPLE_ID_PASSWORD }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-
-  # web:
-  #   name: Build and deploy Web
-  #   needs: validateActor
-  #   if: ${{ fromJSON(needs.validateActor.outputs.IS_TEAM_MEMBER) }}
-  #   runs-on: ubuntu-latest
-  #   steps:
-  #     - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8
-  #       with:
-  #         fetch-depth: 0
-  #         ref: ${{ github.event.pull_request.head.sha }}
-
-  #     - uses: Expensify/App/.github/actions/composite/setupNode@main
-
-  #     - name: Configure AWS Credentials
-  #       # Version: 1.5.5
-  #       uses: aws-actions/configure-aws-credentials@e97d7fbc8e0e5af69631c13daa0f4b5a8d88165b
-  #       with:
-  #         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  #         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  #         aws-region: us-east-1
```

Comment:
As mentioned in Slack, I am pretty sure this code is required anytime you want to communicate with S3, so it's probably required in all jobs (iOS, Android, Desktop).

Reply:
Can we please extract it into a composite run steps action like `setupAWS` in `.github/actions/composite`?

Reply:
It's only one step, do we still want to make it a composite?

Reply:
I'd vote yes, because it reduces duplication and makes other workflows more readable.